app/views/auctions/multi_bid/_win_header.html.erb

```diff
@@ -27,7 +27,7 @@
     </div>
   </div>
   <% end %>
-<% elsif auction.available? && auction.bids? && current_user && auction.user_is_bidder?(current_user) %>
+<% elsif auction.available? && auction.bids? && current_user && auction.user_is_bidder? %>
```

Comment:
I would use this oppty to wrap this into one method on the view model. Also, probabyl the current_user check isn't necessary. Would hope it is inside the #user_is_bidder? method